ipaclient/remote_plugins/__init__.py

```diff
@@ -40,9 +40,14 @@ def __init__(self, api):
     def _read(self):
         try:
             with open(self._path, 'r') as sc:
-                self._dict = json.load(sc)
-        except EnvironmentError as e:
-            if e.errno != errno.ENOENT:
+                self._dict = json.loads(sc)
```

Comment:
Yes, stupid me. I was curious how the file looks like so I had there:
data = sc.read()
json.loads(data)
and printed the data when the error occurred. Obviously, I haven't changed it back properly.
Thanks!